cupsfilters/image.c

```diff
@@ -871,3 +874,121 @@ get_tile(cf_image_t *img,		/* I - Image */
 
   return (ic->pixels + bpp * (y * CF_TILE_SIZE + x));
 }
+
+/*
+  helper function required by EXIF read function
+  */
+
+void trim_spaces(char *buf)
+{
+  char *s = buf - 1;
+  for (; *buf; ++buf)
+  {
+    if (*buf != ' ')
+      s = buf;
+  }
+  *++s = 0; /* null terminate the string on the first of the final spaces */
+}
+
+/*
+  implementation for EXIF read function
+  */
+
+/*
+  helper function to extract bytes from image files
+  */
+
+char *findBytes(FILE *fp, int *size)
```

Comment:
Use `static char find_bytes(...)` here, as this function is local to this file. Also add this function to the overview comment at the top of the file.

Reply:
Especially also note our naming policy, file-local functions are named in all lower case with `_` as separator.